src/passwordValidation.js

```diff
@@ -1,17 +1,21 @@
 export const forbiddenPasswords = ["amG84h6yeQ", "mc9Q20pdjH", "jnT6Q2f8U5"];
 
-export default function isValidPassword(password) {
-  if (typeof password !== 'string') password = String(password.sort((a,b)=>a-b));
-   const regex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z0-9]{10}$/;
-  if(regex.test(password) && !forbiddenPasswords.includes(password) ) {
-    if(new Set([...password]).size > 3) return true
+const isValidPassword = (password) => {
+  if (!typeof password === "string") {
+    return String(password.sort((a, b) => a - b))
   }
-    return false 
-   
-}
-
-
-
-
 
+  const regex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)[a-zA-Z0-9]{10}$/
+  if (forbiddenPasswords.includes(password)) return false
+  if (!regex.test(password)) return false
+  if (!new Set([...password]).size > 3) return false
+  if (password < 4) return false
```

Comment:
I think you can remove this if condition because i don't think it is doing any work instead it can lead to the false result of the function.